src/Stache/Indexes/Value.php

```diff
@@ -30,14 +45,26 @@ public function getItemValue($item)
             return $item->value('authors');
         }
 
+        if (is_array($item)) {
+            return $item[$name] ?? null;
+        }
+
+        if (is_scalar($item)) {
+            return null;
+        }
+
+        if ($item instanceof stdClass) {
+            return $item->$name ?? null;
+        }
```

Comment:
`is_scalar` covers the situation where you attempt to get a child (bar) from a parent (foo) that doesn't have children. Otherwise it was throwing an error for me when testing.

`stdClass` I'll remove - I was just trying to be thorough and cover anything I thought people might throw at it.